style.css

```diff
@@ -1,3 +1,140 @@
-#root {
-  color: red;
+@import url("https://fonts.googleapis.com/css2?family=Baloo+Bhai+2&family=Open+Sans&display=swap");
+
+body {
+  margin: 0 auto;
+  background-color: #000;
+}
+header {
+  display: flex;
+  flex-direction: column;
+  flex-wrap: wrap;
+  align-content: center;
+  align-items: center;
+}
+
```

Comment:
for media queries, it is better at first write down for your mobile screen without any media at the top and after that added media for iPad and laptop with min-width, Lucy gave me this advice 🥇